src/main/java/com/zufar/icedlatte/security/configuration/SpringSecurityConfiguration.java

```diff
@@ -38,6 +38,7 @@ public SecurityFilterChain securityFilterChain(final HttpSecurity httpSecurity,
                                 .requestMatchers(SecurityConstants.FAVOURITES_URL).authenticated()
                                 .requestMatchers(SecurityConstants.ORDERS_URL).authenticated()
                                 .requestMatchers(SecurityConstants.REVIEWS_URL).authenticated()
+                                .requestMatchers(SecurityConstants.RATING_URL).authenticated()
```

Comment:
`GET` should be available for any user, consider using `.requestMatchers(HttpMethod.GET, SecurityConstants.RATING_URL)`